src/client/js/Panels/MetaEditor/MetaEditorControl.js

```diff
@@ -1275,6 +1330,9 @@ define(['js/logger',
         filterIcon = MetaRelations.createButtonIcon(16, MetaRelations.META_RELATIONS.INHERITANCE);
         this.diagramDesigner.addFilterItem('Inheritance', MetaRelations.META_RELATIONS.INHERITANCE, filterIcon);
 
+        filterIcon = MetaRelations.createButtonIcon(16, MetaRelations.META_RELATIONS.MIXIN);
```

Comment:
Can we extract the number 16 magic number? At least put it at the beginning of this function or file.